src/stylesheets/_variables.scss

```diff
@@ -0,0 +1,82 @@
+// Colors
+$grays: (
+  100: #f9fbfd,
+  200: #eef2f5,
+  300: #e6e9ed,
+  400: #c3c6ce,
+  500: #9ea1ac,
+  600: #747784,
+  700: #4d505b,
+  800: #3b3d47,
+  900: #292a33,
+  backgound: #15151a
+);
+
+$reds: (
+  primary: #e20041
+);
+
+$blues: (
+  light: #007aff,
+  primary: #0041e2
+);
+
+$white: #fff;
+$black: #000;
+$red: map-get($reds, 'primary');
+$blue: map-get($blues, 'primary');
+$gray: map-get($grays, 500);
+
+$primary: $white;
+$secondary: $gray;
+
+$link-color: map-get($blues, 'light');
+
+// Fonts
+$font-sizes: (
+  xl: 2.125rem,           // 34px
+  lg: 1.25rem,            // 20px
+  md: 1.0625rem,          // 17px
+  sm: 0.9375rem,          // 15px
+  xs: 0.8125rem,          // 13px
+  xxs: 0.6875rem          // 11px
+);
+
+$font-size-base: map-get($font-sizes, 'md');
+$font-size-lg: map-get($font-sizes, 'lg');
+$font-size-sm: map-get($font-sizes, 'sm');
+
+$font-weight-normal: 400;
+$font-weight-bold: 800;
+
+$h2-font-size: map-get($font-sizes, 'xl');
+$headings-font-weight: $font-weight-bold;
+
+$display1-size: 2.5rem;       // 40px
+$display2-size: 1.75rem;      // 28px
+
+$display1-weight: $font-weight-bold;
+$display2-weight: $font-weight-bold;
```

Comment:
@carryall Its same value as `$display1-weight`, can you use different value?

Reply:
@junan  Actually, I just want to override the Bootstrap variable here, and the value in Figma is also the same one 🤔 
![Screen Shot 2563-11-18 at 15 39 23](https://user-images.githubusercontent.com/1772999/99506162-52817500-29b4-11eb-8342-e10cf236b1f2.png)
![Screen Shot 2563-11-18 at 15 38 24](https://user-images.githubusercontent.com/1772999/99506203-60cf9100-29b4-11eb-988a-10aa625b1142.png)
![Screen Shot 2563-11-18 at 15 38 33](https://user-images.githubusercontent.com/1772999/99506207-6331eb00-29b4-11eb-9f65-4e2078b775fd.png)

Reply:
@carryall What of we create another `_override_bootstrap_variables.scss` and place it after the main `_variables.scss`?
It'll look more easier to manage it 🤔

Reply:
✅  separate the file in [16cb735](https://github.com/carryall/survey-challenge/pull/2/commits/16cb735c3e01d450a83b200ace3880f02e2f1634)